user-guide/Advanced_Modules/DOM/tutorials/DOM_Remove_FieldDescriptor_Definition.md

```diff
@@ -0,0 +1,237 @@
+---
+uid: DOM_Remove_FieldDescriptor_Definition
+---
+
+# Removing a field descriptor from a definition
+
+This tutorial shows how you can remove a field descriptor from a section definition. It uses an example of an existing DOM model with a phone number field that is no longer needed and therefore needs to be removed.
+
+Estimated duration: 10 minutes.
+
+> [!NOTE]
+> The content and screenshots for this tutorial have been created in DataMiner version 10.3.12.
+
+## Prerequisites
+
+- DataMiner version 10.3.12 or higher
+
+- A DataMiner System with an [indexing database](xref:Indexing_Database)
+
+- Basic knowledge of DataMiner Object Models (DOM)
+
+## Overview
+
+- [Step 1: DOM setup](#step-1-dom-setup)
+
+- [Step 2: Soft-delete the field](#step-2-soft-delete-the-field)
+
+- [Step 3: Remove the field from existing instances](#step-3-remove-the-field-from-existing-instances)
+
+- [Step 4: Delete the field descriptor from the section definition](#step-4-delete-the-field-descriptor-from-the-section-definition)
+
+## Step 1: DOM setup
+
+This tutorial assumes that you use the setup from the tutorial [Creating a basic DOM setup](xref:DOM_Create_Basic_Setup), but you can also create a new setup of your own.
```

Comment:
This tutorial is mostly intended for users that have an existing setup where they want to remove a FieldDescriptor. If they just want to try this out, they can create the basic setup. This now seems to indicate that you always need to create something?

Reply:
How about this, in that case?

```suggestion
This tutorial assumes that you use the setup from the tutorial [Creating a basic DOM setup](xref:DOM_Create_Basic_Setup), but you can also use a setup of your own.
```